lib/integrations/base.rb

```diff
@@ -38,5 +38,41 @@ def validate_settings(settings)
         end
       end
     end
+
+    def message_text
+      description = run[:description] ? ": #{run[:description]}" : ""
+      message = self.send(event_name.dup.concat("_message").to_sym)
+      "Your Rainforest Run <#{payload[:frontend_url]} | Run ##{run[:id]}#{description}> #{message}"
```

Comment:
The link format for this is Slack specific, we will need a `href` method or something similar, with, I think, standard HTML link, and then we can override it in Slack, as an example. It would take two arguments, one being the text and the other one the URL.